website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,103 @@
+  </C.Property>
+  <C.Property @name="[RT].PopoverContent" @type="yielded component">
+    Generic container that yields its content inside the "tooltip" popover element.
+    <br />
+    <br />
+    _⚠️ Important: we apply a CSS reset (`all: initial`) to the container, to avoid that styles applied to the parent elements leak into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support)._
+  </C.Property>
+  <C.Property @name="[RT].isOpen" @type="yielded tracked property">
+    Hook into this tracked property to access the state of `isOpen`.
+  </C.Property>
+  <C.Property @name="[RT].close" @type="yielded function">
+    A function to programmatically close the toggle.
+  </C.Property>
+  <C.Property @name="isOpen" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Controls if the tooltip should be rendered initially opened.
+    <br />
+    <br />
+    _Notice: in this case the tooltip can't be dismissed via `esc` or "click outside" until the end user has interacted with it (it's in a "manual" state)._
+  </C.Property>
+  <C.Property @name="isInline" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Sets the display for the HTML element which wraps the "toggle" content. If set to `true` it sets the element's display to `inline-block` (a `<button>` can't have an `inline` layout).
+  </C.Property>
+  <C.Property @name="toggleContainsInteractive" @type="boolean" @values={{(array true false)}} @default={{false}}>
+    Controls if the "toggle" container should be rendered as a `<button>` (default) or as a `<div>` (when it contains interactive elements) to avoid nested interactive elements (non accessible).
+    <br />
+    <br />
+    _Notice: in this case you can't use "click" event listeners for the toggle, or it may result in unexpected behaviours or non accessible code._
+  </C.Property>
+  <C.Property @name="toggleAriaLabel" @type="string">
+    The `aria-label` value applied to the HTML element that acts as "toggle".
+  </C.Property>
+  <C.Property @name="popoverPlacement" @type="enum" @values={{array "top" "top-start" "top-end" "right" "right-start" "right-end" "bottom" "bottom-start" "bottom-end" "left" "left-start" "left-end"}} @default="bottom">
+    Placement for the preferred starting position of the tooltip relative to the toggle element.
+    <br />
+    <br />
+    _Notice: if `@enableCollision` is set, the popover will automatically shift position to remain visible when near the edges of the screen regardless of the starting placement._
+  </C.Property>
+  <C.Property @name="popoverOffset" @type="number|object" @default="12">
+    A number that represents the “distance” between the tooltip and the toggle element (used to control the relative position of the arrow in relation to the toggle). An object can also be passed, which enables to individually configure different axis.
```

Comment:
```suggestion
    A number that represents the “distance” between the tooltip and the toggle element (used to control the relative position of the arrow in relation to the toggle). An object can also be passed, which enables you to individually configure a different axis.
```

I'm not 100% sure what you're trying to say here, but I think this is missing a "you"?